### PR TITLE
feat(portal): add components, refactor portal

### PR DESCRIPTION
**Pull request by lfbergee**
affects: @fremtind/jkl-accordion-react, @fremtind/jkl-logo-react, @fremtind/jkl-logo,
@fremtind/jkl-message-box-react, @fremtind/jkl-message-box, @fremtind/jkl-radio-button,
@fremtind/jkl-text-input, @fremtind/jkl-typography-react, @fremtind/portal

## 📥 Proposed changes

add doc for logo, messagebox, textarea, textfield, radiobutton, tuypography
add import statement
to code section copyable
refactor of component section of portal

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING](https://github.com/fremtind/jokul/blob/master/CONTRIBUTING.md) doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: portal/src/examples/AccordionExample.tsx [hunk 1]
```diff
@@ -10,3 +18,17 @@ export const accordionExample = `
...
+const AccordionExample = () => (
+    <Example
+        exampleComponents={{ Accordion, AccordionItem }}
+        exampleCode={example}
+        types={[{ name: "Accordion", type: accordionType }, { name: "AccordionItem", type: accordionItemType }]}
+        exampleImport={exampleImport}
+    />
+);
+
+export default AccordionExample;
```
Comment (piofinn) [inline portal/src/examples/AccordionExample.tsx]: Are we starting to over-complicate the examples? This probably needs some explanation for someone writing their first component example. Maybe clarify that examples only need the sample code string to work?

File: portal/src/examples/LoaderExample.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Example } from "../components";
+import { Loader } from "@fremtind/jkl-loader-react";
+import "@fremtind/jkl-loader/loader.min.css";
+// @ts-ignore
+import logoType from "!raw-loader!@fremtind/jkl-loader-react/build/Loader.d.ts";
```
[the remark this answers is not in the document]
Comment (lfbergee) [reply portal/src/examples/LoaderExample.tsx]: jep, it can be just type

File: portal/src/examples/AccordionExample.tsx (hunk 1 above)
Comment (lfbergee) [reply portal/src/examples/AccordionExample.tsx]: Well, yes. However, it's not more complicated now than it used to be, it's just that everything is located in a single file, insted of beeing spread out into two.

Comment (lfbergee) [reply portal/src/examples/AccordionExample.tsx]: But yes, as the functionallity grows, the learing curve also grows. And we should document how to use it. I do think its still quite easy to grasp, and its just plain js, no magic.

File: portal/src/pages/components/accordion.tsx [hunk 3]
```diff
@@ -1,45 +1,22 @@
...
-                        { name: "accordionItem", type: accordionItemType },
-                    ]}
-                />
-            }
-        >
-            <div style={{ margin: "2rem 0" }}>
-                <ReactMarkdown renderers={renderer} source={readmeContents} />
-            </div>
-            <Changelog changelog={changelog} />
+        <Layout title="Trekkspilliste" mainExample={<AccordionExample />}>
```
[the remark this answers is not in the document]
Comment (lfbergee) [reply portal/src/pages/components/accordion.tsx]: The reasoning is the fullscreen layout for component, but we can create a separate ComponentLayout, so all other pages can ignore it

File: portal/src/examples/LoaderExample.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (lfbergee) [reply portal/src/examples/LoaderExample.tsx]: copypasta, should have been loaderType maybe, not logo...

File: portal/src/examples/AccordionExample.tsx (hunk 1 above)
Comment (piofinn) [reply portal/src/examples/AccordionExample.tsx]: Yeah, I don't think we've gone too far, but we're at a point where we should probably keep it in mind when/if we do further enhancements. The bar for documenting should be **really** low 😅

**Pull request merged**